src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -44,27 +48,38 @@ const FluxQueryBuilder: FC = () => {
   return (
     <EditorProvider>
       <SidebarProvider>
-        <div className="flux-query-builder--menu">
-          <Button onClick={clear} text="New Script" icon={IconFont.Plus_New} />
-        </div>
-        <DraggableResizer
-          handleOrientation={Orientation.Vertical}
-          handlePositions={vertical}
-          onChangePositions={setVertical}
+        <FlexBox
+          direction={FlexDirection.Column}
+          justifyContent={JustifyContent.SpaceBetween}
+          alignItems={AlignItems.Stretch}
+          style={{height: '100%'}}
         >
-          <DraggableResizer.Panel>
-            <Schema />
-          </DraggableResizer.Panel>
-          <DraggableResizer.Panel
-            testID="flux-query-builder-middle-panel"
-            className="new-data-explorer-rightside"
+          <div className="flux-query-builder--menu" style={{width: '100%'}}>
```

Comment:
nit: could `width: 100%` move to the `flux-query-builder--menu` class?

Reply:
Yes.

And `width: 100%` is actually not needed. Originally was there before i played around with the flexbox options. The latest flexbox options takes care of the width for this element.

I will remove this entirely.